### PR TITLE
update CRDs with extra annotations, generated with go 1.13

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: hack/deploy_csv.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: openshift-marketplace
 spec:
   configMap: codeready-toolchain-saas-host-operator
-  displayName: Codeready Toolchain Saas Host Operator
+  displayName: codeready toolchain saas host operator
```
Comment (MatousJobanek) [inline hack/deploy_csv.yaml]: This look like another incompatibility of mac :-/

Comment (MatousJobanek) [reply hack/deploy_csv.yaml]: @xcoulon Could you please replace these two lines in api olm_catalog.sh script:
https://github.com/codeready-toolchain/api/blob/master/scripts/olm-catalog.sh#L69-L70
```
LOWERCASE=($(echo ${NAME} | tr '-' ' '))
DISPLAYNAME=${LOWERCASE[*]^}
```
by only one line:
```
DISPLAYNAME=$(echo ${NAME} | tr '-' ' ' | sed -e 's/\b\(.\)/\u\1/g')
```
and try to regenerate again?

Comment (xcoulon) [reply hack/deploy_csv.yaml]: that did not change anything in the generated CRD:
```
$ cat hack/deploy_csv.yaml
# This file was autogenerated by github.com/codeready-toolchain/api/olm-catalog.sh'
---
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: codeready-toolchain-saas-host-operator
  namespace: openshift-marketplace
spec:
  configMap: codeready-toolchain-saas-host-operator
  displayName: codeready toolchain saas host operator
  publisher: Red Hat
  sourceType: internal
---
...
```

Comment (MatousJobanek) [reply hack/deploy_csv.yaml]: just for information - this was the correct way:
```
DISPLAYNAME=$(echo ${NAME} | tr '-' ' ' | awk '{for (i=1;i<=NF;i++) $i=toupper(substr($i,1,1)) substr($i,2)} 1')
```

**Pull request merged**